ReactNativeClient/lib/models/Setting.js

```diff
@@ -886,6 +886,7 @@ Setting.THEME_DARK = 2;
 Setting.THEME_SOLARIZED_LIGHT = 3;
 Setting.THEME_SOLARIZED_DARK = 4;
 Setting.THEME_DRACULA = 5;
+Setting.THEME_NORD = 6;
```

Comment:
Please make sure the theme is only available on desktop (mobile and desktop don't share the same theme files).